bin/release_version.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
```

Comment:
this script takes care of: 
- creating and pushing a tag for the bump
- creating an rvm gemset and installing and gems (mostly for fastlane)
- pushing to cocoapods
- building carthage, zipping output and moving it to a folder for easy upload
- creating a bump branch for the next version
- bumping to the next version and pushing the new branch